tests/TestCase/Utility/XmlTest.php

```diff
@@ -573,7 +573,6 @@ public static function invalidArrayDataProvider(): array
             [[]],
             [['numeric key as root']],
             [['item1' => '', 'item2' => '']],
-            [['items' => ['item1', 'item2']]],
```

Comment:
This caused [the following warning](https://github.com/cakephp/cakephp/actions/runs/12224365265/job/34097019729#step:13:259)

Reply:
The point of this provider is to test with invalid values, so this shouldn't be removed.